obspy/signal/spectral_estimation.py

```diff
@@ -2184,23 +2207,27 @@ def __plot_coverage(self, ax):
         ax.autoscale_view()
 
 
-def get_nlnm():
+def get_nlnm(mod_fl):
```

Comment:
I don't think that there was a test for that method done before, could you add it?

Reply:
I removed this argument per megies's advice. Also working-on a new test.